ui/app/templates/jobs/job/variables.hbs

```diff
@@ -0,0 +1,84 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+~}}
+
+{{page-title "Job " @model.job.name " variables"}}
+<JobSubnav @job={{@model.job}} />
+
+<section class="section">
+
+<header class="job-variables-intro">
+  <Hds::Alert @type="inline" @color="highlight" as |A|>
+    <A.Title>Automatic Access to Variables</A.Title>
+    <A.Description>
+      <p>Tasks in this job can have <a href="https://developer.hashicorp.com/nomad/docs/concepts/variables#task-access-to-variables" target="_blank" rel="noopener noreferrer">automatic access to Nomad Variables</a>.</p>
+      <ul>
+        <li>Use 
+          <code>
+            <EditableVariableLink @path="nomad/jobs" @existingPaths={{this.jobRelevantVariables.files}} @namespace={{this.model.job.namespace.name}} />
+          </code>
+           to grant all your jobs access</li>
+        <li>
+          Use
+          <code>
+            <EditableVariableLink @path="nomad/jobs/{{this.model.job.name}}" @existingPaths={{this.jobRelevantVariables.files}} @namespace={{this.model.job.namespace.name}} />
+          </code>
+          to grant this job access
+        </li>
+        <li>
+          Use
+          {{#if (gt this.firstFewTaskGroupNames.length 1)}}
+            {{#each this.firstFewTaskGroupNames as |name|}}
+              <code><EditableVariableLink @path="nomad/jobs/{{this.model.job.name}}/{{name}}" @existingPaths={{this.jobRelevantVariables.files}} @namespace={{this.model.job.namespace.name}} /></code>,
+            {{/each}}
+            etc. to grant a specific task group access
+          {{else}}
+            <code>
+              <EditableVariableLink @path="nomad/jobs/{{this.model.job.name}}/{{object-at 0 this.firstFewTaskGroupNames}}" @existingPaths={{this.jobRelevantVariables.files}} @namespace={{this.model.job.namespace.name}} />
+            </code>
+            to grant your task group access
+          {{/if}}
+        </li>
+        <li>
+          Use
+          {{#if (gt this.firstFewTaskNames.length 1)}}
+            {{#each this.firstFewTaskNames as |name|}}
+              <code><EditableVariableLink @path="nomad/jobs/{{this.model.job.name}}/{{name}}" @existingPaths={{this.jobRelevantVariables.files}} @namespace={{this.model.job.namespace.name}} /></code>,
+            {{/each}}
+            etc. to grant a specific task access
+          {{else}}
+            <code>
+              <EditableVariableLink @path="nomad/jobs/{{this.model.job.name}}/{{object-at 0 this.firstFewTaskNames}}" @existingPaths={{this.jobRelevantVariables.files}} @namespace={{this.model.job.namespace.name}} />
+            </code> to grant your task access
+          {{/if}}
```

Comment:
Nice; digging into this. It might not be whether tasks are running or not, but whether tasks exist whatsoever; in Mirage we have this [shallow](https://github.com/hashicorp/nomad/blob/de71cc2f726802c86a0303d44d06ffab4a8b1294/ui/mirage/scenarios/default.js#L83) rule that we use to say "This is going to be a pretty heavy job, don't fill out all its tasks/groups/evals/task resources/etc.", Throughout our other mock factories we do a bit of `if (!job.shallow)` checks [before we fill them out](https://github.com/hashicorp/nomad/blob/e1e59455e4e65d5107e4c49f452c7bdb443020aa/ui/mirage/factories/task-group.js#L85-L112) . I suspect that's what's going on here, too.